website/content/docs/connect/config-entries/service-defaults.mdx

```diff
@@ -1557,622 +1737,172 @@ Proxy services must be in transparent proxy mode to configure destinations. Refe
-      type: 'TransparentProxyConfig: <optional>',
-      description: `Controls configuration specific to proxies in transparent mode. Added in v1.10.0.`,
-      children: [
-        {
-          name: 'OutboundListenerPort',
-          type: 'int: "15001"',
-          description: `The port the proxy should listen on for outbound traffic. This must be the port where
-                        outbound application traffic is redirected to.`,
-        },
-        {
-          name: 'DialedDirectly',
-          type: 'bool: false',
-          description: {
-            hcl: `Determines whether this proxy instance's IP address can be dialed directly by transparent proxies.
-              Typically transparent proxies dial upstreams using the "virtual" tagged address, which load balances
-              across instances. Dialing individual instances can be helpful in cases like stateful services such
-              as a database cluster with a leader.`,
-            yaml: `Determines whether the Pod IPs can be dialed directly (versus the Cluster IP).
-              Dialing Pod IPs can be helpful in cases like stateful services such
-              as a database cluster with a leader or with an ingress controller that dials Pod IPs instead of Cluster IPs.`,
-          },
-        },
-      ],
-    },
-    {
-      name: 'Destination',
-      type: 'DestinationConfig: <optional>',
-      description: `Controls configuration specific to destinations through terminating-gateway. Added in v1.13.0.`,
-      children: [
-        {
-          name: 'Addresses',
-          type: 'list<string>: []',
-          description:`List of addresses associated with the destination. This can be a hostname or an IP address.
-          Wildcards are not accepted.`,
-        },
+### Enable request rate limit for a service on all paths
+
+The following example limits the request rate on all paths to `service-foo` to an average of `1000` requests per second but allows up to `1500` temporary concurrent requests.
+
+<CodeTabs tabs={[ "HCL", "Kubernetes YAML", "JSON" ]}>
+
+```hcl
+Kind = "service-defaults"
+Name = "service-foo"
+Protocol = "http"
+
+RateLimit {
+  InstanceLevel {
+    RequestsPerSecond = 1000
+    RequestsMaxBurst = 1500
+  }
+}
+```
+```yaml
+kind: ServiceDefaults
+name: service-foo
+protocol: http
+rateLimit: 
+  instanceLevel:
+    requestsPerSecond: 1000
+    requestsMaxBurst: 1500
+  }
+}
+```
+```json
+{
+  "Kind": "service-defaults",
+  "Name": "service-foo",
+  "Protocol": "http",
+  "RateLimit": {
+    "InstanceLevel": {
+      "RequestsPerSecond": 1000,
+      "RequestsMaxBurst": 1500
+    } 
+  }
+}
+```
+</CodeTabs>
+
+### Enable request rate limit on a prefixed path
+ 
+The following example limits the request rate on all paths to `service-foo` to an average of `1000` requests per second but allows up to `1500` temporary concurrent requests. Request paths that begin with `/admin`, however, are limited to an average of `20` requests per second.
```

Comment:
The defaulting of `RequestsMaxBurst` to `RequestsPerSecond` would mean 20 is the maximum
```suggestion
The following example limits the request rate on all paths to `service-foo` to an average of `1000` requests per second but allows up to `1500` temporary concurrent requests. Request paths that begin with `/admin`, however, are limited to `20` requests per second.
```